index.bs

```diff
@@ -6310,53 +5945,13 @@ IANA "WebAuthn Extension Identifier" registry established by [[!WebAuthn-Registr
-    authenticator
-- Specification Document: Section [[#sctn-simple-txauth-extension]] of this specification
-    <br/><br/>
-- WebAuthn Extension Identifier: txAuthGeneric
-- Description: This [=registration extension=] and [=authentication extension=] allows images to be used as transaction
-    authorization prompts as well. This allows authenticators without a font rendering engine to be used and also supports a
-    richer visual appearance than accomplished with the webauthn.txauth.simple extension.
-- Specification Document: Section [[#sctn-generic-txauth-extension]] of this specification
-    <br/><br/>
-- WebAuthn Extension Identifier: authnSel
-- Description: This [=registration extension=] allows a WebAuthn Relying Party to guide the selection of the authenticator that
-    will be leveraged when creating the credential. It is intended primarily for [=[WRPS]=] that wish to tightly
-    control the experience around credential creation.
-- Specification Document: Section [[#sctn-authenticator-selection-extension]] of this specification
-    <br/><br/>
-- WebAuthn Extension Identifier: exts
-- Description: This [=registration extension=] enables the [=[WRP]=] to determine which extensions the authenticator supports. The
-    extension data is a list (CBOR array) of extension identifiers encoded as UTF-8 Strings. This extension is added
-    automatically by the authenticator. This extension can be added to attestation statements.
-- Specification Document: Section [[#sctn-supported-extensions-extension]] of this specification
-    <br/><br/>
-- WebAuthn Extension Identifier: uvi
-- Description: This [=registration extension=] and [=authentication extension=] enables use of a user verification index. The
-    user verification index is a value uniquely identifying a user verification data record. The UVI data can be used by servers
-    to understand whether an authentication was authorized by the exact same biometric data as the initial key generation. This
-    allows the detection and prevention of "friendly fraud".
-- Specification Document: Section [[#sctn-uvi-extension]] of this specification
-    <br/><br/>
-- WebAuthn Extension Identifier: loc
-- Description: The location [=registration extension=] and [=authentication extension=] provides the [=client device=]'s current
-    location to the [=[WRP]=], if supported by the [=client platform=] and subject to [=user consent=].
-- Specification Document: Section [[#sctn-location-extension]] of this specification
-    <br/><br/>
 - WebAuthn Extension Identifier: uvm
 - Description: This [=registration extension=] and [=authentication extension=] enables use of a user verification method.
     The user verification method extension returns to the [=[WRP]=] which user verification methods (factors) were
     used for the WebAuthn operation.
 - Specification Document: Section [[#sctn-uvm-extension]] of this specification
     <br/><br/>
-- WebAuthn Extension Identifier: credProps
```

Comment:
I don't think this one should be deleted. It's the counterpart to rk=preferred.

Reply:
(Based on discussions, this might be correct if this isn't listed by IANA.)

Reply:
I have added credProps back to the IANA registration.